Resources/Prototypes/Damage/modifier_sets.yml

```diff
@@ -109,12 +109,19 @@
 - type: damageModifierSet
   id: Slime
   coefficients:
-    Blunt: 0.6
-    Slash: 1.2
-    Piercing: 1.2
-    Cold: 1.5
-    Poison: 0.8
+    Blunt: 2 # Lots of blunt, if they do more than 5 damage
+    Slash: 1.5
+    Piercing: 0.5
+    Cold: 1.6
+    Heat: 0.4
+    Poison: -1
     Cellular: 0.2
+    Asphyxiation: 0.9
+    Bloodloss: 1.1
+    Shock: 0.4
+    Radiation: 0.5
+  flatReductions:
+    Blunt: 5
```

Comment:
Check random variance- I believe it applies *after* mods. 
If that's not the case, then this is fine. Otherwise, it should probably be lowered *slightly* just so that a punch from a standard crew does *something* to Slimes.

Reply:
[Variance is before modifiers.](https://github.com/Simple-Station/Parkstation/blob/slimin/Content.Shared/Damage/Systems/DamageableSystem.cs#L179-L200)